### DISCUSSION TITLE
PV-Überschussladen startet generell mit 3PH, wieso nicht 1PH?

### DISCUSSION THREAD

**Discussion by gyroslav** [goecharger/go-eCharger-API-v2#160] [Ideas]
Moin!

Habe jetzt erst seit zwei Wochen ein Auto mit 3PH-Lader, und sehe, dass die Go-E generell immer mit 3PH startet, obwohl nur für 1PH PV-Einspeisung vorhanden ist. Das läuft dann auch, und irgendwann schaltet die Go-E auf 1PH um. 

Baut doch bitte ein, dass die Go-E intern den Phasen-Switch immer an die aktuelle Überschussleistung anpasst, auch wenn kein Auto angeschlossen ist. 

Den PGRID kriegt sie ja dauerhaft von mir alle 5 Sekunden. Also müsste die, solange die verfügbare Gesamtleistung < 4200 Watt, PSM=1 stellen. 

Ich werde mir jetzt erstmal damit helfen, dass ich über meinen Script generell PSM=1 stelle, und sobald car=2 und pgrid+Ladeleistung>4500, auf PSM=0 umstelle.

**Comment by mdecker77** [comment]
Du bekommst ein Vote von mir. 

psm ist übrigens in der API Key Liste nicht dokumentiert.
